playbooks/templates/common/metadata_common.yaml

```diff
@@ -0,0 +1,10 @@
+metadata          :
+  RPC: {{ maas_rpc | default(true) }}
+  RPC_env_identifier: {{ maas_env_identifier | default('unkown')}}
+  RPC_env_identifier_alt: {{ maas_env_identifier_alt | default('unkown')}}
+  RPC_maas_version: {{  lookup('pipe', 'cd .. && git describe --tags --abbrev=0') }}
+  RPC_deploy_date: {{ ansible_date_time.date }}
+  RPC_check_category: {{ check_category | default('Unkown') }}
+  RPC_datacenter: {{ maas_rpc_datacenter | default("CDC") }}
+  RPC_device_number: {{ get_server_numbr | default('unkown')}}
```

Comment:
typos:
`get_server_numbr`
`unkown` (this is in multiple places)

Reply:
Also, how are we defining this per-host? Can we leverage raxmon to strip it off the entityURI?

Reply:
I didn't quite yet figure out how to get the server number yet,  we will need to chat about this.

We can utilize inventory_hostname and use raxmon entity URL for sure.